### PR TITLE
Implement support for PrintPage command

### PR DESCRIPTION
**Pull request by alexandru0-dev**
As title says, this PR is for supporting and exposing the PrintPage command.
The only thing that could be improved is not relying on the webdriver crate for PrintParameters (Tho is a pretty long/complex struct so idk)

Already tested myself and works as expected

### CODE REVIEW

**Review by Vrtgs — CHANGES_REQUESTED** [review]
looks great! but the webdriver dependency is big and we don't use it for now, and so I would like it if you add a

`src/common/print.rs`
```rs
use std::sync::Arc;
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
#[serde(untagged)]
pub enum PrintPageRange {
    Integer(u64),
    Range(Arc<str>),
}

#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
#[serde(default, rename_all = "camelCase")]
pub struct PrintParameters {
    pub orientation: PrintOrientation,
    pub scale: f64,
    pub background: bool,
    pub page: PrintPage,
    pub margin: PrintMargins,
    pub page_ranges: Arc<[PrintPageRange]>,
    pub shrink_to_fit: bool,
}

impl Default for PrintParameters {
    fn default() -> Self {
        PrintParameters {
            orientation: PrintOrientation::default(),
            scale: 1.0,
            background: false,
            page: PrintPage::default(),
            margin: PrintMargins::default(),
            page_ranges: Arc::new([]),
            shrink_to_fit: true,
        }
    }
}

#[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
#[serde(rename_all = "lowercase")]
pub enum PrintOrientation {
    Landscape,
    #[default]
    Portrait,
}

#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
#[serde(default)]
pub struct PrintPage {
    pub width: f64,
    pub height: f64,
}

impl Default for PrintPage {
    fn default() -> Self {
        PrintPage {
            width: 21.59,
            height: 27.94,
        }
    }
}

#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
#[serde(default)]
pub struct PrintMargins {
    pub top: f64,
    pub bottom: f64,
    pub left: f64,
    pub right: f64,
}

impl Default for PrintMargins {
    fn default() -> Self {
        PrintMargins {
            top: 1.0,
            bottom: 1.0,
            left: 1.0,
            right: 1.0,
        }
    }
}
```

**Comment by alexandru0-dev** [comment]
As I assumed, I'm also adding a test for printing in the meantime.
One "issue: that I found is clippy complaining about missing docs, and I'm adding them.
The default page size shouldn't something like A4?

**Review by alexandru0-dev — COMMENTED** [review]
Webdriver dependency remove and included inside the crate, also included docs and a test

**Review by alexandru0-dev — COMMENTED** [review]
Webdriver dependency remove and included inside the crate, also included docs and a test

File: thirtyfour/src/common/command.rs [hunk 1]
```diff
@@ -493,6 +495,10 @@ impl FormatRequestData for Command {
                         "value": typing_data.as_vec(), "text": typing_data.to_string()
                     }))
             }
+            Command::PrintPage(params) => {
+                RequestData::new(Method::POST, format!("/session/{}/print", session_id))
+                    .add_body(serde_json::to_value(params).expect("DioCane"))
```
Comment (Vrtgs) [inline thirtyfour/src/common/command.rs:500]: why is this `.expect("DioCane")`? I mean I just want to know the madness behind it

**Comment by Vrtgs** [comment]
looks gread and I'd be happy to merge this if you resolve my concerns with this

File: thirtyfour/src/common/command.rs (hunk 1 above)
Comment (alexandru0-dev) [reply thirtyfour/src/common/command.rs:500]: AHAHHAHHAJHAH
That's a genuine mistake that I've forgotten to remove while testing 
I'm going to fix it immediately

Comment (Vrtgs) [reply thirtyfour/src/common/command.rs]: Nah it's fineeee I do plenty if things like that, but I'd just not want them to sneak to a public repo everyone is going to use

Comment (alexandru0-dev) [reply thirtyfour/src/common/command.rs]: You're completely right, I completely forgot about it as I was still messing around at the time
Fixed and rebased

Comment (alexandru0-dev) [reply thirtyfour/src/common/command.rs]: It's a profanity btw, so i think deleting any references to it if possible would be ideal

**Review by Vrtgs — APPROVED** [review]
Looks good

**Comment by alexandru0-dev** [comment]
It's linting, i can fix that immediately
btw could you take a look at the last comment on the last review request?

**Comment by Vrtgs** [comment]
fix the lint ci, the rest of the tests are wonky for now

**Pull request merged**